File: 백준/Bronze/11034. 캥거루 세마리2/캥거루 세마리2.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        while (true) {
            try {
                int a = sc.nextInt();
                int b = sc.nextInt();
                int c = sc.nextInt();

                System.out.println(maxMoves(a,b,c));
            } catch (Exception e) {
                break;
            }
        }
        sc.close();
    }

    public static int maxMoves(int a, int b, int c) {
        return Math.max(b-a, c-b) - 1;
    }
}

File: 백준/Bronze/11034. 캥거루 세마리2/캥거루 세마리2.cc
#include <iostream>
#include <algorithm>

using namespace std;

int max_moves(int a, int b, int c) {
    return max(b-a, c-b)-1;
}

int main() {
    int a, b, c;
    
    while (cin >> a >> b >> c) {
        cout << max_moves(a,b,c) << endl;
    }
    
    return 0;
}
